controllers/v1/observationSubmissionsController.js

```diff
@@ -113,29 +115,39 @@ module.exports = class ObservationSubmissions extends Abstract {
 
         observationDocument = observationDocument[0];
 
-        let entityDocument = await entitiesHelper.entityDocuments({
-          _id: req.query.entityId,
-          entityType: observationDocument.entityType
-        }, [
-          "metaInformation",
-          "entityTypeId",
-          "entityType",
-          "registryDetails"
-        ]);
+        let entityResult = [];
+        let filterData = {
+            "id" : req.query.entityId,
+            "type" : observationDocument.entityType
+        };
 
-        if (!entityDocument[0]) {
-          return resolve({ 
-            status: httpStatusCode.bad_request.status, 
-            message: messageConstants.apiResponses.ENTITY_NOT_FOUND
-          });
+        let entitiesDocument = await sunbirdService.learnerLocationSearch( filterData );
+        
+        if ( !entitiesDocument.success || !entitiesDocument.data.response.length > 0 ) {
+            let responseMessage = messageConstants.apiResponses.ENTITY_NOT_FOUND;
+            return resolve({ 
+                status: httpStatusCode.bad_request.status, 
+                message: responseMessage 
```

Comment:
@VISHNUDAS-tunerlabs no need to define the "responseMessage" variable you can directly assign the message value inside resolve